analytics/terraform/spark-k8s-operator/addons.tf

```diff
@@ -784,6 +784,21 @@ module "eks_data_addons" {
     repository_password = data.aws_ecrpublic_authorization_token.token.password
   }
 
+  #---------------------------------------------------------------
+  # Kuberay Operator Add-on
+  #---------------------------------------------------------------
+  enable_kuberay_operator = true
```

Comment:
enable_kuberay_operator = var.enable_kuberay_operator

by default this value is `false`